connectors/atlassian/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "engines": {
-    "node": "10"
-  },
-  "dependencies": {
-      "superagent": "^5.2.2",
-      "@fusebit/add-on-sdk": "^0.0.1"
-  }
-}
\ No newline at end of file
+    "engines": {
+        "node": "10"
+    },
+    "dependencies": {
+        "superagent": "^5.2.2",
+        "@fusebit/add-on-sdk": "^1.0.6"
```

Comment:
1.0.6? 1.3.1?

Reply:
Won't fix, I'm only working on Slack in this change (with the exception of prettifying the other addons)